src/PuliPlugin.php

```diff
@@ -466,10 +466,10 @@ private function loadPuliPackages()
         $packages = array();
 
         $output = $this->puliRunner->run(
-            'package --list --format \'%name%;%installer%;%install_path%;%state%\''
+            'package --list --format "%name%;%installer%;%install_path%;%state%"'
```

Comment:
I've got it. In Windows command line, the quotation mark for surrounded arguments is double quote: [Command-Line Arguments (MSDN doc)](https://msdn.microsoft.com/en-us/library/17w5ykft%28v=vs.85%29.aspx). And when Ilooked at msdn doc, I never saw single quote for surrended arguments ([example](http://blogs.msdn.com/b/oldnewthing/archive/2010/09/17/10063629.aspx)).

So this is not a Symfony Process issue but a Puli one.

I start to update single quote to double, output now correctly match as expected but I still got a failure on tests, if it mean somethink for you:
```
1) Puli\ComposerPlugin\Tests\PuliPluginTest::testInstallNewPuliPackages with data set #0 ('post-install-cmd')
Composer\IO\IOInterface::writeError('<warning>Warning: No root pac...rning>', true) was not expected to be called.

PATH\puli\composer-plugin\src\PuliPlugin.php:511
PATH\puli\composer-plugin\src\PuliPlugin.php:322
PATH\puli\composer-plugin\src\PuliPlugin.php:168
PATH\puli\composer-plugin\tests\PuliPluginTest.php:219

2) Puli\ComposerPlugin\Tests\PuliPluginTest::testInstallNewPuliPackages with data set #1 ('post-update-cmd')
```